Source/SwiftLintBuiltInRules/Rules/Idiomatic/NoEmptyBlockRule.swift

```diff
@@ -0,0 +1,104 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct NoEmptyBlockRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "no_empty_block",
+        name: "No Empty Block",
+        // swiftlint:disable:next line_length
+        description: "Code Blocks(Function bodies, catch, defer) should not be empty; they should at least contain a comment",
```

Comment:
Since this is used as the violations's message (as shown on the console or in IDEs and editors), it should be reasonable short. So I would say the following is sufficient:
```suggestion
        description: "Code blocks should not be empty; they should at least contain a comment",
```